packages/ui/src/common/components/typography/SidePaneLabel.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react'
 import styled from 'styled-components'
 
-interface AboutMemberProps {
+interface Props {
   text: string
   className?: any
 }
 
-export const MembershipLabel = React.memo(({ text, className }: AboutMemberProps) => {
+export const SidePaneLabel = React.memo(({ text, className }: Props) => {
```

Comment:
This one stands out. I think that it should land together with other side-pane components.